app/cafe.py

```diff
@@ -0,0 +1,21 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        NotWearingMaskError,
+                        OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> None:
+        if "vaccine" not in visitor.keys():
+            raise NotVaccinatedError("Visitor is not vaccinated.")
```

Comment:
The error message 'Visitor is not vaccinated.' could be more descriptive. Consider including more context, such as 'Visitor cannot enter the cafe because they are not vaccinated.'